ruby/classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
```

Comment:
- [ ] For the associations between the `Classroom` and the `Student` classes to work properly when trying to invoke the `add_students` method it has to be set up `attr_reader :students` 🙏